toolkit/dump_tasks.php

```diff
@@ -44,27 +44,32 @@
 		exit - 1;
 	}
 
+	echo "iTop: $sITopUrl\n";
 	if (empty($aResult['objects'])) {
-		echo "There is no SynchroDataSource defined on the iTop server ($sITopUrl).\n";
+		echo "   no SynchroDataSource defined\n";
 	} else {
-		switch (count($aResult['objects'])){
+		switch (count($aResult['objects'])) {
 			case 1:
-				echo "There is 1 SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo "   1 SynchroDataSource defined\n";
 				break;
 
 			default:
-				echo "There are ".count($aResult['objects'])." SynchroDataSource defined on the iTop server ($sITopUrl):\n";
+				echo '   '.count($aResult['objects'])." SynchroDataSource objects defined\n";
 				break;
 		}
 
-		echo "+--------------------------------+----------------------------------------------------+\n";
-		echo "|            Name                |                    Description                     |\n";
-		echo "+--------------------------------+----------------------------------------------------+\n";
+		echo "+--------+--------------------------------+----------------------------------------------------+\n";
+		echo "|   Id   |            Name                |                    Description                     |\n";
+		echo "+--------+--------------------------------+----------------------------------------------------+\n";
 		foreach ($aResult['objects'] as $aValues) {
 			$aCurrentTaskDefinition = $aValues['fields'];
-			echo sprintf("| %-30.30s | %-50.50s |\n", $aCurrentTaskDefinition['name'], $aCurrentTaskDefinition['description']);
+			echo sprintf("| %-6.6s | %-30.30s | %-50.50s |\n",
```

Comment:
Right-justify integers might look better..
```suggestion
			echo sprintf("| %6d | %-30.30s | %-50.50s |\n",
```

Reply:
Thanks, nice suggestion ! 
Description updated accordingly